src/test/regress/sql/multi_utilities.sql

```diff
@@ -229,7 +229,9 @@ VACUUM;
 insert into local_vacuum_table select i from generate_series(1,1000000) i;
 delete from local_vacuum_table;
 VACUUM local_vacuum_table;
-SELECT CASE WHEN s BETWEEN 20000000 AND 25000000 THEN 22500000 ELSE s END
+VACUUM local_vacuum_table;
+VACUUM local_vacuum_table;
+SELECT CASE WHEN s BETWEEN 20000000 AND 49999999 THEN 35000000 ELSE s END size
```

Comment:
Why did these sizes change?

Reply:
my apologies, that's a copy paste mistake